scripts/auditdocgen/main.go

```diff
@@ -117,7 +117,8 @@ func updateAuditDoc(doc []byte, auditableResourcesMap AuditableResourcesMap) ([]
 	buffer.WriteString("|--|-----------------|\n")
 
 	for _, resourceName := range sortedResourceNames {
-		buffer.WriteString("|" + resourceName + "|<table><thead><tr><th>Field</th><th>Tracked</th></tr></thead><tbody>")
+		auditActionsString := strings.Join(audit.AuditActionMap[resourceName], ", ")
+		buffer.WriteString("|" + resourceName + "<br><i>" + auditActionsString + "|<table><thead><tr><th>Field</th><th>Tracked</th></tr></thead><tbody>")
```

Comment:
nit: doesn't `<i>` require `</i>`?

Reply:
Good question. It's rendering correctly, but I see it's not identified as self-closing so I'll fix!